CONTRIBUTING.md

```diff
@@ -1,18 +1,26 @@
 # Contributing to Dogecoin Core
 
-The Dogecoin Core project operates an open contributor model where anyone is
-welcome to contribute towards development in the form of peer review, testing
-and patches. This document explains the practical process and guidelines for
-contributing.
+Dogecoin Core is an open source project, and we would welcome contributions which provably
+improve the state of the software. As a pointer on where to start, please look at the
+[projects](https://github.com/dogecoin/dogecoin/projects) currently underway,
```

Comment:
Project link is duplicate (DRY) (also inside community section).
Following my next comment, by reorganizing sections, you can give this info almost directly and in a single place.

Reply:
Restructured, see if that works?